src/app/organization/offices/office-data-tables.resolver.ts

```diff
@@ -0,0 +1,30 @@
+/** Angular Imports */
```

Comment:
There is already an offices data tables resolver in common resolvers folder use that.